### DISCUSSION TITLE
Is it possible to show children on parent name entering?

### DISCUSSION THREAD

**Discussion by SivachenkoYegor** [dowjones/react-dropdown-tree-select#561] [Q&A]
For example, I have a data structure like this (from one of your examples on codeSandbox):
[
  {
    label: "VP Accounting",
    checked: true,
    children: [
      {
        label: "iWay",
        children: [
          { label: "Universidad de Especialidades del Espíritu Santo" },
          { label: "Marmara University" },
          { label: "Baghdad College of Pharmacy" }
        ]
      },
      {
        label: "KDB",
        children: [
          { label: "Latvian University of Agriculture" },
          { label: "Dublin Institute of Technology" }
        ]
      }
    ]
  }
];

In the input field, I'm entering first letters like "KD". Is it possible to show not just "KDB" but its children too?

Thanks

**Comment by RamsayRomero** [comment]
Set the `keepTreeOnSearch` and `keepChildrenOnSearch` props to true
